cmd/frontend/graphqlbackend/org_invitations.go

```diff
@@ -111,6 +114,39 @@ func checkEmail(ctx context.Context, db database.DB, inviteEmail string) (bool,
 	return false, nil
 }
 
+func (r *schemaResolver) PendingInvitations(ctx context.Context, args *struct {
+	Organization graphql.ID
+}) ([]*organizationInvitationResolver, error) {
+	actor := actor.FromContext(ctx)
+	if !actor.IsAuthenticated() {
+		return nil, errors.New("no current user")
+	}
+
+	var orgID int32
+	if err := relay.UnmarshalSpec(args.Organization, &orgID); err != nil {
+		return nil, err
+	}
+
+	// 🚨 SECURITY: Check that the current user is a member of the org that the user is being
+	// invited to.
+	if err := backend.CheckOrgAccessOrSiteAdmin(ctx, r.db, orgID); err != nil {
+		return nil, err
+	}
+
+	pendingInvites, err := r.db.OrgInvitations().GetPendingByOrgID(ctx, orgID)
+
+	if err != nil {
+		return nil, err
+	}
+
+	var invitations []*organizationInvitationResolver
+	for _, invitation := range pendingInvites {
+		invitations = append(invitations, r.toOrganizationInvitation(invitation))
```

Comment:
This should work even without the `toOrganizationInvitation` method 
```suggestion
		invitations = append(invitations, NewOrganizationInvitationResolver(r.db, invitation))
```